cli/src/commit_templater.rs

```diff
@@ -1381,6 +1437,70 @@ impl Template for Vec<Rc<CommitRef>> {
     }
 }
 
+/// Workspace name together with its working-copy commit for templating.
+#[derive(Debug, serde::Serialize)]
+pub struct WorkspaceRef {
+    /// Workspace name as a symbol.
+    name: RefSymbolBuf,
```

Comment:
nit: it's probably better to use `WorkspaceNameBuf` as we know it is the name type.